pkg/components/securedata/securedata.go

```diff
@@ -0,0 +1,26 @@
+package securedata
+
+import (
+	"encoding/base64"
+
+	"github.com/grafana/grafana/pkg/setting"
+	"github.com/grafana/grafana/pkg/util"
+)
+
+type SecureData string
```

Comment:
Might it be more efficient to make this `[]byte` instead, to avoid having to convert to string?